teps/0091-trusted-resources.md

```diff
@@ -377,13 +377,18 @@ metadata:
   name: feature-flag
   namespace: tekton-pipelines
 data:
-  verification-policy: enforce
+  resource-verification-mode: verify_all
 ```
 
-`verification-policy`. (Optional, `enforce`, `warn` or `skip`, default `skip`):
- * `enforce`: Failing verification will mark the taskruns/pipelineruns as failed.
- * `warn`: Log warning but don't fail the taskruns/pipelineruns.
- * `skip`: Directly skip the verification.
+`resource-verification-mode`. (Optional, `verify_all`, `verify_match` or `no_verification`, default `no_verification`):
+ * `verify_all`: Failing verification will mark the taskruns/pipelineruns as failed.
+ * `verify_match`: fetch VerificationPolicy from the namespace, and if policies matched then verify, otherwise skip the verification.
```

Comment:
nit: This could be explained elsewhere, but by match it is not clear what is supposed to match. Maybe a link to matching point 2 in "Verify Resources" could be useful.

Reply:
👍